contracts/contracts/rem.system/src/voting.cpp

```diff
@@ -79,17 +79,28 @@ namespace eosiosystem {
             info.last_votepay_share_update = ct;
          });
          _gstate.total_producer_stake += tot.own_stake_amount;
+
+         const auto &vot = _voters.get(producer.value, "user has no resources");
```

Comment:
@AndriiAndrusyshyn please move this code out of _else_ branch; it's should be applied for both new and existing producer